File: Aula01-introducao-poo/ts/main.ts
import { ParkingTicket } from './ParkingTicket';

const entryTime = new Date(2026, 1, 23, 8, 10, 0);
const exitTime = new Date(2026, 1, 23, 11, 45, 0);

const ticket = new ParkingTicket(entryTime);
ticket.close(exitTime);

const total = ticket.calculateTotal();
console.log(`Total to pay: ${total.toFixed(2)}`);

File: Aula01-introducao-poo/java/Main.java
import java.time.LocalDateTime;

public class Main {
    public static void main(String[] args) {
        LocalDateTime entryTime = LocalDateTime.of(2026, 2, 23, 8, 10);
        LocalDateTime exitTime = LocalDateTime.of(2026, 2, 23, 11, 45);

        ParkingTicket ticket = new ParkingTicket(entryTime);
        ticket.close(exitTime);

        System.out.println("Total to pay: " + ticket.calculateTotal());
    }
}
